### ISSUE TITLE
Mods are not showing in my modpack

### ISSUE THREAD

**Issue by megamanzteam** [gorilla-devs/GDLauncher#445]
All the mods I installed to my mod pack suddenly disappeared from GDLauncher. They do still exist on the hard drive, just not showing in the program. I tried to repair, but it doesn't help. I would like to avoid having to re-install all the mods I added. Suggestions?

**Comment by AzureDoom** [comment]
Did you update forge?

**Comment by megamanzteam** [comment]
No, I just opened the program today and they are no longer showing.

**Comment by furtabs** [comment]
What OS.

GDLauncher has switched directories way more then I can count. If you can make a dummy modpack and find the new spot copy your old packs over.

If your using the old version we don’t support it anyone for bug fixes but you might 
 Have not read the file that said don’t drag mods in

**Comment by megamanzteam** [comment]
I am on Windows 10. My directory is: C:\Users\[USERNAME]\AppData\Roaming\GDLauncher\instances\packs. I did a new dummy and it built the modpack in the same location as the one in question.

> If your using the old version we don’t support it anyone for bug fixes but you might
> Have not read the file that said don’t drag mods in

I have only ever installed mods from within the launcher.

I took a look (not edit) at config.json and the mods section is blank. How could I quickly re-build the file to reflect the mods that were installed?
